include/service/entities/site_service.js

```diff
@@ -598,13 +602,16 @@ module.exports = function SiteServiceModule(pb) {
      * @return {Object}
      */
     SiteService.getGlobalSiteContext = function() {
+        var defaultLocale = pb.Localization.getDefaultLocale();
+        var defaultSupportedLocales = {};
+        defaultSupportedLocales[defaultLocale] = true;
         return {
             displayName: pb.config.siteName,
             uid: pb.SiteService.GLOBAL_SITE,
             hostname: pb.config.multisite.enabled ? url.parse(pb.config.multisite.globalRoot).host : url.parse(pb.config.siteRoot).host,
             active: pb.config.multisite.enabled ? false : true,
-            defaultLocale: pb.Localization.getDefaultLocale(),
-            supportedLocales: util.arrayToObj(pb.Localization.getSupported(), function(a, i) { return a[i]; }, function(a, i) { return true; }),
+            defaultLocale: defaultLocale,
+            supportedLocales: defaultSupportedLocales,
```

Comment:
Agreed. This should be an object.  Using a single locale for the global site isn't what we want to do.  The line that was removed asks localization for an array of Locale objects.  It then runs that through the ```util.arrayToObj``` which allows for converting an array to a hash.  You can run it through the debugger but I'm pretty sure that it provides an object as expected.